### PR TITLE
[MOB-4036] messages not displayed upon tapping

### PR DESCRIPTION
**Pull request by evangreer91**
## 🔹 JIRA Ticket(s) if any

* [MOB-4036](https://iterable.atlassian.net/browse/MOB-4036)

## ✏️ Description

This pull request introduces the SafeAreaView component for the iterableInbox component. This pull request also adds the logic to turn off the pan responder when the user is clicking.

### CODE REVIEW

File: ts/IterableInbox.tsx [hunk 1]
```diff
@@ -64,21 +66,52 @@ const IterableInbox = ({
...
-      headline
+      headline,
+      messageListContainer
    } = styles
 
    const navTitleHeight = headline.height + headline.paddingTop + headline.paddingBottom
-   const updatedContainer = { ...container, width: 2 * width, height: height - navTitleHeight - 40 }
-   const messageListContainer = { width: width }
-
-   headline = { ...headline, height: Platform.OS === "android" ? 70 : 60 }
```
Comment (Ayyanchira) [inline ts/IterableInbox.tsx:77]: Title cutting off because of this

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
```
Comment (Ayyanchira) [inline .vscode/launch.json:13]: Add gitignore to skip this file

**Review by Ayyanchira — COMMENTED** [review]
Added some comments

File: ts/IterableInbox.tsx (hunk 1 above)
Comment (evangreer91) [reply ts/IterableInbox.tsx:77]: Got it adding it back

**Review by Ayyanchira — APPROVED** [review]
Looks good 👍

**Pull request merged**